src/app/recents/recents.component.css

```diff
@@ -0,0 +1,7 @@
+.recent-title {
+    margin-left: 6rem;
```

Comment:
Instead of duplicating this style, could you wrap the header and the tabs in a div and apply the style on the div? Also, `6rem` is too much on mobile mode. Additionally, in mobile mode, the right side of the table hugs the window frame. How about modeling this after the `estuary-container` CSS class, including the media queries?